bin/seed.js

```diff
@@ -471,6 +472,29 @@ const init = async () => {
                 Macadamias,
             ]),
         ])
+
+        const seedReviews = async () => {
+            const puffReview = await Review.writeNew(
+                alejandra.id,
+                Puff.id,
+                5,
+                'Totally addicted!'
+            )
+            const pineappleCakeReview = await Review.writeNew(
+                kevin.id,
+                PineappleCake.id,
+                5,
+                'So good!'
+            )
+            const iceCreamBarReview = await Review.writeNew(
+                yiru.id,
+                IceCreamBar.id,
+                5,
+                'Yum! Will definitely be ordering again!'
+            )
+        }
+        seedUsers()
```

Comment:
Would have to think about it, but these might need to be await?

Reply:
I'll fix that.